### DISCUSSION TITLE
Integrating UI with API - Gauge/Taiko

### DISCUSSION THREAD

**Discussion by vidhyav20** [getgauge/taiko#2576] [General]
I am working on Integrating UI Testing Framework with API Testing to achieve a more reliable Automation Framework.
Hence need assistance on Integrating REST API Testing with existing Gauge & Taiko framework. Is this possible? Any ideas?

**Comment by zabil** [comment]
Taiko and Gauge js runs on Node. As Taiko is only for testing browsers, you'll have use another node library for testing REST Api's with Gauge js.

**Comment by gulshankamboj** [comment]
Yes, you can use Supertest or any other node js library for API testing and integrate that with Gauge framework. It works flawlessly. You can use any node library. 
PS: I was here 6 months back (Glad to help!!!)
